### PR TITLE
Display:none on share block (between caption and pictures) in amp

### PR DESCRIPTION
**Pull request by NataliaLKB**
fixing: #11966

### CODE REVIEW

File: common/app/views/fragments/amp/stylesheets/main.scala.html [hunk 1]
```diff
@@ -246,6 +246,12 @@
         list-style: none;
     }
 
+    @* temporarily hiding these for after launch *@
```
Comment (jamesgorrie) [inline common/app/views/fragments/amp/stylesheets/main.scala.html]: until after launch?

**Comment by jamesgorrie** [comment]
:+1:

File: common/app/views/fragments/amp/stylesheets/main.scala.html (hunk 1 above)
Comment (NataliaLKB) [inline common/app/views/fragments/amp/stylesheets/main.scala.html]: Haha yes... Thanks you!

File: common/app/views/fragments/amp/stylesheets/main.scala.html [hunk 2]
```diff
@@ -246,6 +246,12 @@
...
+    .inline-expand-image, .content__dateline-lm , .block-share, .witness-cta-wrapper, .block-share {
+        display: none;
+        visibility: hidden;
```
Comment (OliverJAsh) [inline common/app/views/fragments/amp/stylesheets/main.scala.html]: You shouldn't need both? `display: none` removes the node from the render tree entirely.

Comment (NataliaLKB) [inline common/app/views/fragments/amp/stylesheets/main.scala.html]: Force of habit. Thanks @OliverJAsh :smile:

**Pull request merged**